addons/web_editor/static/src/js/editor/rte.summernote.js

```diff
@@ -1074,8 +1074,14 @@ var SummernoteManager = Class.extend(mixins.EventDispatcherMixin, {
             return;
         }
         data.__alreadyDone = true;
+
         var mediaDialog = new weWidgets.MediaDialog(this,
-            data.options || {},
+            _.extend({
+                domain: ['|',
+                    '&', ['res_model', '=', data.$editable.data('oe-model')], ['res_id', '=', data.$editable.data('oe-id')],
+                    ['res_model', '=', 'ir.ui.view']],
```

Comment:
Why is this needed ?

Reply:
To have access token on the website and load image from attachments

Reply:
Now, you can have image on the product displayed on the website, and the widget display the attachments on the website and we should not put them on the website anymore. What is easier for users to navigate and share documents, data sheets...